src/IonSpan.ts

```diff
@@ -51,6 +51,14 @@ export abstract class Span {
 
   protected abstract clone(start: number, len: number): Span;
 
+  locator(pos: number) : number[] {
+    throw new Error("not implemented");
```

Comment:
Did you mean for these methods to be `abstract` instead of throwing a not-implemented error?